### ISSUE TITLE
Lower Cardinality Nuances

### ISSUE THREAD

**Issue by cmoesel** [standardhealth/shr-grammar#4]
From @markkramerus:

In CAMEO 5.0, we need to capture some nuances about the lower cardinality bound. Here are the cases I can think of:
 
1. The element it is simply optional
2. The element can be null, but if you have the data, you must provide it (e.g. “Must Support”)
3. The element must be present, but it is ok if the value is missing, as long as you provide an explanation for the null value
4. The element cannot be missing and its value cannot be missing, full stop.
 
Basically, 0 and 1 don’t cut it. We need four lower cardinality values, e.g., 0, 0S, 1E, 1

**Comment by markkramerus** [comment]
The combination of MustSupport and cardinality is now the operative standard for interpreting the lower cardinality bound. Closing.
